tools/appimage.sh

```diff
@@ -385,10 +385,6 @@ cd ./$APP/
 # but not using the host system's information about what is
 # installed on the system but our own assumptions instead
 
-mkdir -p ./tmp/archives/
-mkdir -p ./tmp/lists/partial
-touch tmp/pkgcache.bin tmp/srcpkgcache.bin
```

Comment:
In case you wonder why they are removed -- same operations are done in the `generate_status` function called below.